AdobeMediaGalleryApi/Model/Asset/Command/SaveInterface.php

```diff
@@ -11,19 +11,18 @@
 use Magento\AdobeMediaGalleryApi\Api\Data\AssetInterface;
 
 /**
- * Save media asset.
- *
+ * Save media asset
  * @api
```

Comment:
Could you please remove all the `@api` annotation from `AdobeMediaGalleryApi` module. We cannot add API to the patch release, but we can mark these interfaces back with `@api` later in 2.4 release.

I missed this point initially, but thankfully that was brought up by @joni-jones during the public meeting yesterday